opencti-platform/opencti-front/src/private/components/settings/users/edition/UserEditionConfidence.tsx

```diff
@@ -187,10 +187,10 @@ const UserEditionConfidence: FunctionComponent<UserEditionConfidenceProps> = ({
                   <IconButton
                     color="primary"
                     aria-label="Add"
-                    onClick={() => arrayHelpers.push({ entity_type: '', max_confidence: 0 })}
+                    onClick={() => arrayHelpers.push({ entity_type: '', max_confidence: user.effective_confidence_level && user.effective_confidence_level.max_confidence ? user.effective_confidence_level.max_confidence : 0 })}
```

Comment:
Badass cette ligne de code

Could be a function or at least separate into multiple lines?